capellambse/helpers.py

```diff
@@ -367,6 +367,40 @@ def repair_html(markup: str) -> markupsafe.Markup:
     markup
         The repaired markup.
     """
+
+    def cb(node: etree._Element) -> None:
+        for k in list(node.keys()):
+            if ":" in k:
+                del node.attrib[k]
+
+    return handle_html(markup, cb)
+
+
+def handle_html(
+    markup: str, node_handler: cabc.Callable[[etree._Element], None]
+) -> markupsafe.Markup:
+    """Modify HTML markup using a provided node handler function.
```

Comment:
The docstring needs to mention these important factoids:

- This accepts and returns an HTML fragment (i.e. there doesn't have to be a surrounding root element)
- More than the first element handed to the callback may have no parent (implied by the previous point, but IMO we should highlight it by mentioning it explicitly)
- The callback won't be called for leading text if there is any, and so has no ability to influence it